.azure-pipelines/test_plan.py

```diff
@@ -344,8 +421,12 @@ def poll(self, test_plan_id, interval=60, timeout=36000):
                 scripts=get_test_scripts(args.test_set),
                 output=args.output
             )
-        elif args.action == "poll":
-            tp.poll(args.test_plan_id, args.interval, args.timeout)
+        elif args.action == "lock":
+            tp.lock(args.test_plan_id, args.interval, args.timeout)
+        elif args.action == "prepare":
+            tp.prepare(args.test_plan_id, args.interval, args.timeout)
+        elif args.action == "running":
+            tp.running(args.test_plan_id, args.interval, args.timeout)
```

Comment:
It looks like the code here is to `lock`, `prepare` and `run` test plan according to the `args.action` value. But the corresponding methods of the `tp` object are just to get the test plan status to see if they are in the expected state. I am confused here...